cl/clparams/initial_state/initial_state.go

```diff
@@ -68,6 +68,10 @@ func GetGenesisState(network clparams.NetworkType) (*state.CachingBeaconState, e
 		if err := returnState.DecodeSSZ(gnosisStateSSZ, int(clparams.Phase0Version)); err != nil {
 			return nil, err
 		}
+	case clparams.ChiadoNetwork:
+		if err := returnState.DecodeSSZ(gnosisStateSSZ, int(clparams.Phase0Version)); err != nil {
```

Comment:
@shotasilagadze actually I just realised - are you sure that `gnosisStateSSZ` here is the right one? Isn't it for gnosis mainnet? There is one for Chiado here - https://github.com/gnosischain/configs/blob/main/chiado/genesis.ssz maybe that is the one?

Reply:
yep, getting below error after I did this https://github.com/erigontech/erigon/pull/12701
```
INFO] [11-11|16:37:19.511] [Caplin] starting clstages loop          app=caplin
[INFO] [11-11|16:37:19.511] Starting downloading History             app=caplin stage=DownloadHistoricalBlocks from=13189312
[DBUG] [11-11|16:37:19.515] request failed                           app=downloader error="Head \"https://erigon2-v1-snapshots-chiado.erigon.network/manifest.txt\": dial tcp: lookup erigon2-v1-snapshots-chiado.erigon.network on 127.0.0.53:53: no such host" method=HEAD url=https://erigon2-v1-snapshots-chiado.erigon.network/manifest.txt
[DBUG] [11-11|16:37:19.885] Gotten unexpected root                   got=0xd566eb4ba463ffbfd9bbe503d295cb5f28ba86fa90f700e2f0d7582dbc8a3814 expected=0x3af6a8f94d7d81e513eeb3a94d8ccda9a4d7d3f1de1e38b5e4c12cd6b4496346
```